### PR TITLE
Added support for service.version

### PR DESCRIPTION
**Pull request by tobiasstadler**
See #132. For the reference, I did not have a look at the implementation in #132.

### CODE REVIEW

**Comment by tobiasstadler** [comment]
@eyalkoren I will add support for it in the agent once this is released.

**Comment by tobiasstadler** [comment]
@eyalkoren Would you please have a look?

**Comment by tobiasstadler** [comment]
ping

**Comment by eyalkoren** [comment]
@tobiasstadler sorry, we are quite overloaded atm, I hope we have a chance to get to this and additional ecs-logging stuff soon enough 🙏

File: docs/tab-widgets/ecs-encoder.asciidoc [hunk 1]
```diff
@@ -53,7 +54,12 @@ All you have to do is to use the `co.elastic.logging.logback.EcsEncoder` instead
 |`serviceName`
 |String
 |
-|Sets the `service.name` field so you can filter your logs by a particular service
+|Sets the `service.name` fieldso you can filter your logs by a particular service name
```
Comment (eyalkoren) [inline docs/tab-widgets/ecs-encoder.asciidoc:57]:
```suggestion
|Sets the `service.name` field so you can filter your logs by a particular service name
```

File: docs/tab-widgets/ecs-encoder.asciidoc [hunk 2]
```diff
@@ -53,7 +54,12 @@ All you have to do is to use the `co.elastic.logging.logback.EcsEncoder` instead
...
+
+|`serviceVersion`
+|String
+|
+|Sets the `service.version` fieldso you can filter your logs by a particular service version
```
Comment (eyalkoren) [inline docs/tab-widgets/ecs-encoder.asciidoc:62]:
```suggestion
|Sets the `service.version` field so you can filter your logs by a particular service version
```

File: jul-ecs-formatter/src/main/java/co/elastic/logging/jul/EcsFormatter.java [hunk 3]
```diff
@@ -96,6 +99,10 @@ protected void setServiceName(final String serviceName) {
         this.serviceName = serviceName;
     }
 
+    protected void setServiceVersion(final String serviceVersion) {
```
Comment (eyalkoren) [inline jul-ecs-formatter/src/main/java/co/elastic/logging/jul/EcsFormatter.java:102]:
```suggestion
    public void setServiceVersion(final String serviceVersion) {
```
The other shouldn't be `protected` as well, fixed through #179

**Review by eyalkoren — COMMENTED** [review]
@tobiasstadler Apologies for the long wait 🙏 
Few very minor comments.
Please just bring up to date, then we'll test and merge.
Thanks for your patience!

**Review by eyalkoren — APPROVED** [review]

**Review by felixbarny — APPROVED** [review]

**Comment by tobiasstadler** [comment]
Thank You!

**Pull request merged**